client/block.c

```diff
@@ -1445,9 +1445,8 @@ static int bi_compar(const void *l, const void *r)
 }
 
 // returns string representation for the block state. Ignores BI_OURS flag
-static const char* get_block_state_info(struct block_internal *block)
+const char* xdag_get_block_state_info(uint8_t flag)
 {
-	const uint8_t flag = block->flags & ~BI_OURS;
```

Comment:
I would recommend you to leave this line. 
`& ~BI_OURS` on each call of function does not looke good.